tests/expr_and_series/len_test.py

```diff
@@ -6,7 +6,18 @@
 from tests.utils import compare_dicts
 
 
-def test_len(constructor: Any, request: Any) -> None:
+def test_len_no_filter(constructor: Any) -> None:
+    data = {"a": list("xyz"), "b": [1, 2, 1]}
+    expected = {"l": [3], "l2": [6]}
+    df = nw.from_native(constructor(data)).select(
+        nw.col("a").len().alias("l"),
+        (nw.col("a").len() * 2).alias("l2"),
+    )
+
+    compare_dicts(df, expected)
+
+
+def test_len_len_chaining(constructor: Any, request: Any) -> None:
```

Comment:
Renamed this test, and keep failing for dask as `.filter` is not implemented. Wondering if that's ok to implement even in the sad case that we do not move forward with #743 . 

At the end, I think `expr.filter` is mainly use to filter within a context and then aggregate (exactly as in the test here)